src/app/models/media-wall-custom.ts

```diff
@@ -0,0 +1,69 @@
+export interface WallHeader {
+	backgroundColor: string;
+	fontColor: string;
+}
+
+export interface WallBackground {
+	backgroundColor: string;
+}
+
+export interface WallGrid {
+	columnCount: string;
+	columnGap: string;
+}
+
+export interface WallCard {
+	fontSize: string;
+	fontColor: string;
+	backgroundColor: string;
+	accentColor: string;
+}
+
+/**
+ * @function defaultColumnCount
+ * Checks the current screen size and returns the default column count
+ *
+ * @return string
+ */
+
+export function defaultColumnCount(): string {
```

Comment:
Using flex-box creates an issue of vertical gap. If we use flex boxes, gaps are produced depending on the maximum height of a card in a row. Using column count can make item align vertically without any gaps.
That is why column count would be a good approach for this.